lib/stsci/tools/teal.py

```diff
@@ -310,8 +310,8 @@ def _isInstalled(fullFname):
     try:
         import site
         instAreas = site.getsitepackages()
-    except:
-        pass # python 2.6 and lower don't have site.getsitepackages()
+    except Exception:
+        instAreas = []  # python 2.6 and lower don't have site.getsitepackages()
```

Comment:
Do we still support 2.6?

Reply:
I am going to remove PY2 entirely in the next PR.